recipes/tokyocabinet/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+./configure --prefix=$PREFIX \
+	    --with-zlib=$PREFIX \
+	    --with-bzip=$PREFIX
+
+make
+make check > /dev/null
```

Comment:
I looked for a button like this, but only found it for circleCI. I like your solution. Done now.